src/main/webapp/Clothing.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Food</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="listing-container">
+      <h3 class="title">Clothes Donations</h3>
+
+      <div class="listing">
+        <div class="listing-left">
+        </div>
+
+        <div class="listing-right">
+          <h3 class="listing-title">Adults clothes Pick UP ASAP!</h3>
+
+          <div class="listing-description">
+              Hello my name is "Name" and I'm here to tell you
+              that I have 3 jackets and 4 pants to donate. Adults size only.
+            <ul>
+              <li>Zipcode: 00000</li>
+              <li>Posted: 1 hour ago</li>
+            </ul>  
+          </div>
+
+          <div class="listing-chat">
+              <a href="">Chat</a>
+          </div>
+          
+        </div>
+      </div>
+    </div>
+
+  </body>
+</html>
```

Comment:
Spacing and indentation. We need to decide if we want 4 space indentation or 2 space.

Reply:
I did 2 spaces for all of them

Reply:
Make sure you chat with the team and come to an agreement so we don't have inconsistent spacing in sections that different people have worked on